src/docfx/serve/lsp/TextDocumentHandler.cs

```diff
@@ -16,20 +17,42 @@ namespace Microsoft.Docs.Build
 {
     internal class TextDocumentHandler : ITextDocumentSyncHandler
     {
-        private readonly Channel<FileActionEvent> _channel;
-        private readonly DocumentSelector _documentSelector = new(new DocumentFilter { Pattern = "**/*.{md,yml,json}" });
+        private static readonly string[] s_configFileNames =
```

Comment:
is it possible to not specialize config?

Reply:
for what reason?